Candy helper.js

```diff
@@ -0,0 +1,33 @@
+const boughtCandyPrices =[];
+const candyPrices = {
```

Comment:
This exercise is very well implemented, good job! It's also nice that you have covered the case for invalid candy types.

There is a common convention for [**truly immutable** constants](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage) (I know it sounds confusing since `const` is already immutable, right?) to have all uppercase letters. This is very opinionated, even not part of the language standard, but something that many dev teams adopt in their code style. Normally, we would use this naming for something that is declared at the global level, for example math constants (`PI`, `FIRST_PRIME_NUMBER` etc.) or some predefined data.

Again, this isn't required, just wanted to share this with you!

```suggestion
const CANDY_PRICE = {
```